src/main/assets/js/session-timeout.ts

```diff
@@ -41,6 +40,17 @@ const pingUserActive = throttle(
   { trailing: false }
 );
 
+const getSessionTimeoutInterval = (): number => {
+  const timeoutParam = new URL(location.href).searchParams.get('timeout');
+  if (timeoutParam && !isNaN(parseInt(timeoutParam))) {
+    return parseInt(timeoutParam);
+  }
+
+  return [WEBCHAT_URL, TIMED_OUT_URL].includes(window.location.pathname as PageLink)
+    ? 12 * 60 * 60 * 1000 // 12 hours
```

Comment:
why does the timedOut url result in a 12hr timeout?

Reply:
Spoke to Michael about this - he wants timeout page to show indefinitely so the user can see that they've been signed out for privacy reasons and their data has been saved. The way it works atm is that if the user was to leave their PC for over 40mins, they would then see IDAM page with no explanation on why they have logged out. 
It's safe to leave them on timeout page indefinitely because their session has already been destroyed.